components/blitz/src/loci/ome/io/OmeroReader.java

```diff
@@ -470,7 +469,7 @@ else if (key.equals("iid")) {
     public static void main(String[] args) throws Exception {
         try (BufferedReader con = new BufferedReader(
                 new InputStreamReader(System.in, Constants.ENCODING));
-                OmeroReader omeroReader = new OmeroReader()) {
+             OmeroReader omeroReader = new OmeroReader()) {
```

Comment:
I kept the indent for ``InputStreamReader`` since it was initialized as part of ``BufferedReader``
I found it easier to understand
happy to change if you think it does not add any value